app/uk/gov/hmrc/exports/mongock/changesets/CacheChangelog.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmrc.exports.mongock.changesets;
+
+import com.github.cloudyrock.mongock.ChangeLog;
+import com.github.cloudyrock.mongock.ChangeSet;
+import com.mongodb.client.MongoDatabase;
+
+@ChangeLog
+public class CacheChangelog {
+    private String collection = "declarations";
```

Comment:
What about other collections? (Notifications and Submissions)

Reply:
We don't need for now to change those collections. If necessary we will just add it.